### PR TITLE
Fixed margins between elements in paginator

### LINKED ISSUE #1358: Paginator with incorrect margins
**Describe the bug**
There are inconsistencies between Paginator and its design specifications. Margins between elements are not correct.

**To Reproduce**
Design: 
image

Paginator component:
image

**Expected behaviour**
It should have a correct margins as it is specified in the design.

**Add labels**
Design, guidelines, paginator.

### PR DESCRIPTION
**Pull request by Lisa159**
**Checklist**
_(Check off all the items before submitting)_
- [x] Build process is done without errors and all tests pass in `/lib` directory.
- [x] Self-reviewed the code prior to submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Purpose**
Paginator has wrong margins between elements. 

**Description**
Fixed values in tokens and added missing margins.

**Closes #1358**

### CODE REVIEW

File: docs/src/pages/themeBuilder/themes/AdvancedTheme.json [hunk 1]
```diff
@@ -495,13 +495,13 @@
     "fontTextTransform": "none",
     "verticalPadding": "0.75rem",
     "horizontalPadding": "2rem",
-    "marginRight": "40px",
-    "marginLeft": "20px",
+    "marginRight": "2rem",
+    "marginLeft": "2rem",
```
Comment (marcialfps) [inline docs/src/pages/themeBuilder/themes/AdvancedTheme.json:499]: What is the reason for this change?

Comment (Lisa159) [reply docs/src/pages/themeBuilder/themes/AdvancedTheme.json]: Undone

**Review by marcialfps — APPROVED** [review]
